### DISCUSSION TITLE
How to run the script on a website that is loaded inside an iframe?

### DISCUSSION THREAD

**Discussion by italodeandra** [quoid/userscripts#608] [Q&A]
I created the UserScript with `// @include` being the URL that is on the iframe. But when I open the parent page on Safari, it doesn't load the script. Is it not supported by iPhone Safari? When using on my PC it works normally.

**Comment by ACTCD** [comment]
It works the same way as macOS, I tested it on iOS and it injects `iframe` properly.

Try checking the relevant logs: https://github.com/quoid/userscripts/issues/409
